src/Features/SupportModels/EloquentCollectionSynth.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Livewire\Features\SupportModels;
+
+use Illuminate\Database\Eloquent\Collection as EloquentCollection;
+use Illuminate\Database\Eloquent\Relations\Relation;
+use Illuminate\Queue\SerializesAndRestoresModelIdentifiers;
+use Livewire\Mechanisms\HandleComponents\Synthesizers\Synth;
+
+class EloquentCollectionSynth extends Synth {
+    use SerializesAndRestoresModelIdentifiers;
+
+    public static $key = 'elcln';
+
+    static function match($target)
+    {
+        return $target instanceof EloquentCollection;
+    }
+
+    function dehydrate(EloquentCollection $target, $dehydrateChild)
+    {
+        $class = $target::class;
+        $modelClass = $target->getQueueableClass();
```

Comment:
`getQueueableClass` return null if the collection is empty. 
Then it throws error when calling`$modelAlias = (new $modelClass)->getMorphClass();`.

I'm not comfortable myself with Synths, maybe @joshhanley you have an idea on how to fix this ?

Reply:
@calebporzio I always wonder if you would prefer to discuss newly introduced bug in the MR or in a new Discussion ?

Reply:
I'm working on a fix myself

Reply:
I submitted a PR #7017, tests are passing on my machine.

Reply:
For anyone else seeing this, fix has been released in v3.0.7.